dspace-server-webapp/src/test/java/org/dspace/app/rest/authorization/EnrollAdministratorIT.java

```diff
@@ -0,0 +1,101 @@
+ */
+package org.dspace.app.rest.authorization;
+
+import static org.springframework.data.rest.webmvc.RestMediaTypes.TEXT_URI_LIST_VALUE;
+import static org.springframework.http.MediaType.parseMediaType;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.dspace.app.rest.authorization.impl.AdministratorFeature;
+import org.dspace.app.rest.builder.EPersonBuilder;
+import org.dspace.app.rest.converter.ConverterService;
+import org.dspace.app.rest.matcher.AuthorizationMatcher;
+import org.dspace.app.rest.model.SiteRest;
+import org.dspace.app.rest.projection.DefaultProjection;
+import org.dspace.app.rest.test.AbstractControllerIntegrationTest;
+import org.dspace.content.Site;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.SiteService;
+import org.dspace.eperson.EPerson;
+import org.dspace.eperson.Group;
+import org.dspace.eperson.service.GroupService;
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * test that the enroll process for a new administrator, i.e. create an user and
+ * add him to the administrators group, will result in the Administrator Feature(s) to be available to such user.
+ * 
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk at 4science.it)
+ *
+ */
+public class EnrollAdministratorIT extends AbstractControllerIntegrationTest {
+
+    @Autowired
+    private AuthorizationFeatureService authorizationFeatureService;
+    @Autowired
+    private ConverterService converterService;
+    @Autowired
+    GroupService groupService;
+
+    private SiteService siteService;
+
+    /** 
+     * this hold a reference to the test feature {@link AdministratorFeature}
+     */
+    private AuthorizationFeature administratorFeature;
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        siteService = ContentServiceFactory.getInstance().getSiteService();
+        administratorFeature = authorizationFeatureService.find(AdministratorFeature.NAME);
+    }
+
+    @Test
+    public void addUserToAdminGroupOnSiteTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        EPerson eperson1 = EPersonBuilder.createEPerson(context)
+                          .withNameInMetadata("Jhon", "Brown")
+                          .withEmail("johnbrown@example.com")
+                          .withPassword(password)
+                          .build();
+
+        context.restoreAuthSystemState();
+
+        Site site = siteService.findSite(context);
+        SiteRest siteRest = converterService.toRest(site, DefaultProjection.DEFAULT);
```

Comment:
Just a note. A lot of these Converters get refactored/renamed in the (currently open) PR #2726.  So, depending on which PR is merged first (this one or that one), the other will require refactoring slightly.